### PR TITLE
UTR: add unit test for EIP_721_ALL

### PR DESCRIPTION
**Pull request by anhnq82**

### CODE REVIEW

File: contracts/UniversalTokenRouter.sol [hunk 1]
```diff
@@ -137,7 +138,12 @@ contract UniversalTokenRouter is IUniversalTokenRouter {
             if (token.id == EIP_721_ALL) {
                 return IERC721(token.adr).balanceOf(owner);
             }
-            return IERC721(token.adr).ownerOf(token.id) == owner ? 1 : 0;
+            try IERC721(token.adr).ownerOf(token.id) {
```
Comment (Zergity) [inline contracts/UniversalTokenRouter.sol:141]: why do we need this? if the id is invalid, it still revert, right?

**Review by Zergity — CHANGES_REQUESTED** [review]

File: contracts/UniversalTokenRouter.sol (hunk 1 above)
Comment (anhnq82) [reply contracts/UniversalTokenRouter.sol:141]: if ID doesn't exist, IERC721(token.adr).ownerOf(token.id) will throw an error and revert tx
so I put try cacth to return 0 for _balanceOf function in this case

Comment (Zergity) [reply contracts/UniversalTokenRouter.sol:141]: no need for `ERC721_INVALID_TOKEN_ID` check

**Review by Zergity — CHANGES_REQUESTED** [review]

**Review by Zergity — CHANGES_REQUESTED** [review]
please add an UT for a specific 721 id

**Review by Zergity — APPROVED** [review]

**Pull request merged**